src/widgets/formula/content-view.js

```diff
@@ -47,18 +49,27 @@ module.exports = cdb.core.View.extend({
     var prefix = this.model.get('prefix');
     var suffix = this.model.get('suffix');
 
+    var sourceId = this._dataviewModel.get('source').id;
+    var letter = layerColors.letter(sourceId);
+    var sourceColor = layerColors.getColorForLetter(letter);
+
     this.$el.html(
       template({
         title: this.model.get('title'),
+        sourceId: sourceId,
+        sourceType: analyses.title(this._dataviewModel.getSourceType()),
```

Comment:
I'd add `|| ''` if it doesn't exist, I guess it could prevent to show an `undefined`